robber/matchers/exception.py

```diff
@@ -0,0 +1,33 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class ExceptionMatcher(Base):
+    """
+    expect(lambda: call_something(with_some_params)).to.throw(any_exception)
+    """
+    @property
+    def _raised(self):
+        raised = None
+
+        if not callable(self.actual):
+            raise TypeError('{actual} is not callable'.format(actual=self.actual))
+
+        try:
+            self.actual()
+        except BaseException as raised:
+            return raised
+
+    def matches(self):
+        return isinstance(self._raised, self.expected)
```

Comment:
If the use-case is valid, I guess we can add syntax like:
```python
expect(callable).to.throw.exactly(this_exception)
```
but for this, I still prefer to keep it (since the most popular one like `sure` is still alive with it :P).